docs/protocol-tutorial/0010-states-channels.md

```diff
@@ -0,0 +1,305 @@
+        uint48 challengeDuration;
+    }
+
+    struct VariablePart {
+        Outcome.SingleAssetExit[] outcome; // (1)
+        bytes appData;
+        uint48 turnNum;
+        bool isFinal;
+    }
+    ```
+
+    1. This composite type is explained in the section on [outcomes](./0030-outcomes.md).
+
+=== "TypeScript"
+
+    ```typescript
+        import * as ExitFormat from '@statechannels/exit-format';
+        import {Address, Bytes, Bytes32, Uint256, Uint48, Uint64} from '@statechannels/nitro-protocol'; // (1)
+
+        export interface FixedPart {
+            chainId: Uint256;
+            participants: Address[];
+            channelNonce: Uint64;
+            appDefinition: Address;
+            challengeDuration: Uint48;
+        }
+
+        export interface VariablePart {
+            outcome: ExitFormat.Exit; // (2)
+            appData: Bytes;
+            turnNum: Uint48;
+            isFinal: boolean;
+        }
+    ```
+
+    1. `Bytes32`, `Bytes`, `Address`, `Uint256`, `Uint64` are aliases to the Javascript `string` type. They are respresented as hex strings. `Uint48` is aliased to a `number`.
+    2. This composite type is explained in the section on [outcomes](./0030-outcomes.md).
+
+=== "Go"
+
+    ```Go
+    import (
+        "github.com/statechannels/go-nitro/channel/state/outcome"
+        "github.com/statechannels/go-nitro/types" // (1)
+    )
+
+    type (
+        FixedPart struct {
+            ChainId           *types.Uint256
+            Participants      []types.Address
+            ChannelNonce      uint64
+            AppDefinition     types.Address
+            ChallengeDuration uint32
+        }
+
+        VariablePart struct {
+            AppData types.Bytes
+            Outcome outcome.Exit // (2)
+            TurnNum uint64
+            IsFinal bool
+        }
+    )
+    ```
+
+    1. `types.Address` is an alias to go-ethereum's [`common.Address`](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.10.8/common#Address) type. `types.Bytes32` is an alias to go-ethereum's [`common.Hash`](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.10.8/common#Hash) type.
+    2. This composite type is explained in the section on [outcomes](./0030-outcomes.md).
+
+!!! info
+
+    States are usually submitted to the blockchain as a single fixed part and multiple (signed) variable parts. This is known as a ["support proof"](#support-proofs).
+
+Let's take each property in turn:
+
+### Chain id
+
+This needs to match the id of the chain where assets are to be locked (i.e. the 'root' of the funding graph for this channel). In the event of a mismatch, the channel cannot be concluded and funds cannot be unlocked.
+
+### Participants
+
+This is a dynamic array of Ethereum addresses, each derived from an ECDSA private key in the usual manner. Each address represents a participant in the state channel who is able to commit to state updates and thereby cause the channel to finalize on chain.
```

Comment:
Dynamic?

Reply:
dynamically sized array? https://docs.soliditylang.org/en/develop/types.html#arrays

Reply:
That description is solidity specific (go would refer to a slice, TS just an array). Given that the type definitions are immediately above in each language, we can probably drop that level of technical description in favour of "A collection of Ethereum addresses, ...".